index.html

```diff
@@ -3270,38 +3270,17 @@ <h2 id="incubations">
       </h2>
       <p>
         Extensions to this specification are being incubated in parallel by the
-        Web Community, some of which are shipping in multiple browsers. If two
-        or more browser engines end up supporting an incubated feature, then
-        those features will become part of this specification in the future -
-        allowing them to become a standard the Web Platform:
+        Web Community, some of which are shipping in multiple browsers. All
+        incubations can be found in the <a href=
+        "https://github.com/WICG/manifest-incubations">Manifest Incubations
+        repository</a> at the <a href="https://wicg.io"><abbr title=
+        "Web Incubator Community Group">WICG</abbr></a>.
+      </p>
+      <p>
+        If two or more browser engines end up supporting an incubated feature,
+        then those features will become part of this specification in the
+        future - allowing them to become a standard of the Web Platform.
```

Comment:
Can you link too the charter, particularly to scope? 

```suggestion
        then those features can be eligible to become part of this specification in the
        future (so long as they are in scope of the Working Group's charter) - allowing them to become a standard of the Web Platform.
```